common/src/main/java/io/crate/common/StringUtils.java

```diff
@@ -24,11 +24,31 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 import javax.annotation.Nullable;
 
 public final class StringUtils {
 
+    public static String toString(Class<?> clazz, Object...attributes) {
+        if (attributes.length % 2 != 0) {
+            throw new IllegalArgumentException("expected event number of entries");
+        }
+        StringBuilder sb = new StringBuilder(32);
+        sb.append(clazz.getClass().getName()).append('{');
+        if (attributes.length > 0) {
+            for (int i = 0; i <= attributes.length / 2; i += 2) {
+                sb.append(Objects.requireNonNull(attributes[i]))
+                    .append("=")
+                    .append(attributes[i + 1])
+                    .append(", ");
+            }
+            sb.setLength(sb.length() - 2);
+        }
+        sb.append("}");
```

Comment:
I don't think that this complexity is really worth it for simple toString methods. Also, most IDE's have nice toString method generators.